src/index.tsx

```diff
@@ -249,6 +281,9 @@ export function generateTrigger(
 
     onMouseEnter = e => {
       const { mouseEnterDelay } = this.props;
+      if ((e.target as any).disabled) {
```

Comment:
target may dynamic change  to disabled

Reply:
@zombieJ you mean it will change to/from disabled `onMouseEnter`?

Reply:
1. onMouseEnter: Popup
2. code set to disabled - miss